### PR TITLE
Insights on EMR

### PR DESCRIPTION
**Pull request by melvinsoft**

### CODE REVIEW

**Comment by melvinsoft** [comment]
[(8) Insights EMR: Create Ansible playbook to deploy analytics components to run pipelines](https://trello.com/c/PZMjKKsv/2249-8-insights-emr-create-ansible-playbook-to-deploy-analytics-components-to-run-pipelines)

File: playbooks/roles/jenkins_master/handlers/main.yml [hunk 1]
```diff
@@ -3,13 +3,22 @@
   service:
     name: jenkins
     state: restarted
+  tags:
```
Comment (tkeemon) [inline playbooks/roles/jenkins_master/handlers/main.yml]: What's the purpose of adding tags to a handler? My understanding is that the handler will be run whenever a tasks contains `notify`. Do the tags change this behavior somehow?

Comment (melvinsoft) [reply playbooks/roles/jenkins_master/handlers/main.yml]: This has been introduced when I pulled last upstream changes here. Maybe is to be able to perform small changes, that aren't hooked up with the handler.

File: playbooks/appsemblerPlaybooks/ficus_analytics_jenkins.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- name: Install python2 on EC2 instance
+  hosts: jenkins-scheduler
+  sudo: yes
+  remote_user: ubuntu
+  gather_facts: no
+  pre_tasks:
+    - name: "Update apt-get"
+      raw: sudo apt-get update -y # && sudo apt-get upgrade
+    - name: "install python2"
```
[the remark this answers is not in the document]
Comment (melvinsoft) [reply playbooks/appsemblerPlaybooks/ficus_analytics_jenkins.yml]: Done!

**Comment by melvinsoft** [comment]
@tkeemon Thanks for your review, merging now.

**Pull request merged**